README.md

```diff
@@ -3,7 +3,7 @@
 # My Portfilio
```

Comment:
### **Optional changes :** 📌

Figma version        | Your Mobile version
-------------------- | ----------------------
![template](https://user-images.githubusercontent.com/54015740/143611626-d9b12056-432d-4d39-a57d-eebab05b17f9.png) | ![Mobile](https://user-images.githubusercontent.com/54015740/143719266-c7a24b54-960c-4080-bc40-58ba50ab396b.png)

> if we compare both versions, the Figma version at the 100% percent shows the icons of the social media, while yours in 100% doesn't show them, and this is due to the dimensions you used on your CSS,  kindly change the dimensions to make it fit and look like in the template example, to make your project look better, since this feature is highly important for professional reasons.

### Recommendations :
- Kindly make sure to always check how your project fits on 100% when looking at it on mobile version.
---
![coding](https://user-images.githubusercontent.com/54015740/143619782-76a5258d-0a82-4cb3-b07a-645f6f9c1355.gif)
---
###  Happy coding 🎉😄